common/src/Microsoft.Azure.IIoT.Core/src/Utils/JsonConvertEx.cs

```diff
@@ -57,7 +57,11 @@ public static class JsonConvertEx {
         /// <returns></returns>
         internal static JsonSerializerSettings GetDefaultSettings() {
             return new JsonSerializerSettings {
-                ContractResolver = new DefaultContractResolver(),
+                ContractResolver = new DefaultContractResolver() {
+                    NamingStrategy = new CamelCaseNamingStrategy {
+                        ProcessDictionaryKeys = false
```

Comment:
QQ: where is this coming into play?

Reply:
this is required for the serialization of the payload dictionary if form "mykey": {"value":"myvalue"}

Reply:
Got it, thanks!